spec/common/providers/https.spec.ts

```diff
@@ -172,35 +220,6 @@ function checkAppCheckContext(
   expect(context.instanceIdToken).to.be.undefined;
 }
 
-function checkAuthRequest(
```

Comment:
Deleted these because a CallableRequest _is_ a CallableContext. Made the overload explicit in the method I kept anyway.